src/openrct2/object/ObjectRepository.cpp

```diff
@@ -98,13 +98,12 @@ class ObjectRepository final : public IObjectRepository
     std::vector<ObjectRepositoryItem>   _items;
     QueryDirectoryResult                _queryDirectoryResult = { 0 };
     ObjectEntryMap                      _itemMap;
-    uint16                              _languageId = 0;
-    sint32                                 _numConflicts;
+    uint16                              _languageId   = 0;
+    sint32                              _numConflicts = 0;
 
 public:
-    ObjectRepository(IPlatformEnvironment * env)
+    ObjectRepository(IPlatformEnvironment * env) : _env(env)
     {
-        _env = env;
     }
```

Comment:
It has to discard that because your logic for setting those read-only fields could be quite complicated - many lines long. The constructor is all about setting up the object and that includes setting all its read-only fields.